renovate.json5

```
@@ -0,0 +1,78 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
+  ],
+  "timezone": "Asia/Tokyo",
+  "separateMajorMinor": false,
+  "dependencyDashboardApproval": true, // 万が一`packageRules`の網羅性に穴ができた場合に備え
+  "packageRules": [
+    {
```

多分ルールが複雑化してきてパッとわからなくなってくると思うので、コメントで分かりやすくしておくのどうでしょう！
なんか数年後の僕たちの集中力をちょっと奪うかもと思ったので。

ここはこうとか。
```suggestion
    // メジャーバージョンの更新
    {
```